### PR TITLE
[Bugfix:Submission] Inconsistent behaviour while uploading zip file

### LINKED ISSUE #4833: Dialog box for uploading zip inconsistent with expected behavior
**Describe the bug**
When I drag a zip into submission box, I get a dialog box that says all files in the bucket will be deleted if I upload the zip, and do I want to continue. Hitting cancel on that dialogue adds the zip to the bucket and leaves all of my files in-place. My expectation is that hitting cancel would neither remove any files from the bucket, nor add the zip to the bucket.

**Expected behavior**
Either the dialog should be made more clear on what will happen if you hit cancel. I'd expect either one of the two to happen:
1. Hitting cancel will not delete any previously uploaded files in the bucket, but will also add the zip (as it currently does)
1. Hitting cancel will cause the zip to not be uploaded.

where if it's the first case, the dialog message should be made more clear, and if it's the second case, we should fix the behavior (and maybe reword the dialog box slightly?).

**To Reproduce**
1. Put files into bucket
2. Drag zip file into bucket
3. See Dialogue
1. Hit cancel

**Configuration**
 - OS: [e.g. Windows, Linux, ...] macOS
 - Browser [e.g. chrome, safari] Chrome

**Screenshots**
Screenshot 2019-12-03 06 16 41
Screenshot 2019-12-03 06 16 47

**Additional context**
Before any PR gets opened, it should be determined in this issue what exactly the bug is.

The zip does not contain any files that would match the other files in the bucket.

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)

### What is the current behavior?
Closes #4833 

### What is the new behavior?
Bug is now fixed and respective feature is working as per the expectation in the mentioned [issue](https://github.com/Submitty/Submitty/issues/4833)

### CODE REVIEW

**Comment by mukulkrjha** [comment]
@MasterOdin Hey PTAL!

**Comment by MasterOdin** [comment]
@mukulkrjhaThanks for the PR. I needed to confirm with @bmcutler that this was the desired behavior before merging.
